bzl/bazel.rc

```diff
@@ -32,11 +32,13 @@ test --test_env=HOME
 test --test_env=PLAIDML_DEVICE
 test --test_env=PLAIDML_TARGET
 test --test_env=PLAIDML_SETTINGS
+test --test_env=PLAIDML_VERBOSE
 test --test_sharding_strategy=disabled
 run --test_env=HOME
 run --test_env=PLAIDML_DEVICE
 run --test_env=PLAIDML_TARGET
 run --test_env=PLAIDML_SETTINGS
+run --test_env=PLAIDML_VERBOSE
```

Comment:
Why do we need this? I've been using PLAIDML_VERBOSE without a need to add this to the .bazelrc for a long time now.

Reply:
Interesting; I'd expect bazel to have cleaned it from your test environment.  IIRC it doesn't work without something like this, but I can remove it and add it to my .bazelrc.user (assuming it's needed) too.

Reply:
It's not needed AFAICT, was it not working for you in some context? I'd like to understand what prompted you to add this.

Reply:
I think we have the sandbox disabled BTW

Reply:
I actually added it in my local codebase ages ago, for some reason; I've just been carrying it forward.  I'll try removing it along with all the other environment passthroughs.

Reply:
AFAIK, no changes are needed to the `.bazelrc`, or at least no changes that are related to this PR :)